### DISCUSSION TITLE
save to csv

### DISCUSSION THREAD

**Discussion by miaxhee** [cozodb/cozo#14] [Q&A] [answered]
Hi,

Thanks for this amazing work!

Q: How can I export the result of a ? rule or a stored relation into e.g. csv or someting else? Is there something like CsvWrite()?

**Comment by zh217** [comment, marked as answer]
You need to use your host language to convert the results to CSV. Cozo does not write to disk, except to the database file.

**Reply by miaxhee** [reply]
ok, I see. 
Thanks!
